packages/near-fast-auth-signer/src/components/SignMultichain/schema.ts

```diff
@@ -0,0 +1,6 @@
+import { BorshSchema } from 'borsher';
+
+export const derivationPathSchema = BorshSchema.Struct({
+  asset:         BorshSchema.String,
```

Comment:
I'm not sure if it should be a string. 

I think it's better create a enum of supported chains and re-use it on all the places where you do chain checks.